File: java101/Alapok/2017.10.14 stringek/jelszo/src/jelszo/Jelszo.java
package jelszo;
import java.util.Random;
public class Jelszo {

    public static void main(String[] args) {
        String kar = "abcdefghijklmnopqrstuvwxyz";
        kar += kar.toUpperCase();
        kar += "0123456789_()";
        Random vg = new Random();
        int vel;
        String jelszo;
        
        for (int i = 0; i < 5; i++) {
            jelszo = "";
            for (int j = 0; j < 16; j++) {
                vel = vg.nextInt(kar.length());
                jelszo += kar.charAt(vel);
            }
            System.out.println(jelszo);
        }
    }
}

File: python101/python2/11 Stringek 2/Megoldások/jelszo.py
# Jelszó generátor program
import random
kar = 'abcdefghijklmnopqrstuvwxyz'
kar += kar.upper()
kar += '0123456789'
for i in range(5):
    jelszo = ''
    for j in range(10):
        jelszo += random.choice(kar)
    print(jelszo)

